autogpt_platform/backend/backend/integrations/credentials_store.py

```diff
@@ -225,6 +234,8 @@ def get_all_creds(self, user_id: str) -> list[Credentials]:
         all_credentials.append(ollama_credentials)
 
         # These will only be added if the API key is set
+        if settings.secrets.example_api_key:
```

Comment:
Forgot this but these need added to the frontend too to hide from the integrations page

Reply:
Actually the provider needs it too

Reply:
The number of files that need to be modified for adding a new block / integration is concerning me.

Do you think we could reduce that requirement?